### ISSUE TITLE
Loss function of VAE

### ISSUE THREAD

**Issue by blastlove** [jsxlei/SCALEX#27]
Hi,  thanks for your developing this useful tool! I noticed that you used binary cross entropy rather than mean square error as the reconstruction loss term.

 I learned VAE from the textbook, and the derivation of reconstruction term is as below.
<img width="475" alt="image" src="https://github.com/jsxlei/SCALEX/assets/111514867/1c2f7d09-c6fa-46d6-8760-ce3f636bc394">

The above formula shows that the reconstruction term is the mean square error between x and recon_x. So I'm curious why you chose to use binary cross entropy. 

Thanks a lot if you can reply to this question.

**Comment by jsxlei** [comment]
Hi thanks for your interests. Since we min max the values range from 0 to 1, it will be a liitle better to use binary cross entropy here to optimize the model. Mean square error also works.
